sass/popup.scss

```diff
@@ -66,7 +66,8 @@ svg {
   }
 }
 
-#i-menu {
+#i-menu,
+#i-plus {
```

Comment:
I think it may be better to align the button to the same spot that the scan button is since the action is similar.

Reply:
Updated SASS to put this in the same spot as the scan button